earth_enterprise/src/fusion/rasterfuse/gepackgen.cpp

```diff
@@ -85,6 +89,9 @@ int main(int argc, char *argv[]) {
   if (!imagery && !terrain) {
     usage(progname, "--imagery or --terrain is required");
   }
+  if (imagery && (decimation_threshold != 0.009)) {
```

Comment:
This is really picky, but it is possible for someone to specify `--imagery --decimation_threshold=0.009` and avoid this validation check.  You could set the decimation threshold to -1 by default and only set it to 0.009 after doing this check.  I'll leave that up to you - I'm happy to approve this either way.

Reply:
Well, it's a valid point, although having decimation set while generating imagery packets doesn't seem to have any effect .  I skimmed khGetOpt.h earlier for other validation possibilities and I can do it like this:  options.setExclusive("imagery", "decimation")

Reply:
Although now I sort of want to do that to some of the other exclusive option pairs .. I tested and it seems useful

Reply:
As I said, it's up to you, though the setExclusive thing does seem handy.

Reply:
sample output from using setExclusive():

$ ./gepackgen --imagery --decimation 0.1234
Error: The following options are exclusive:
	--decimation
	--imagery

usage: ./gepackgen [options] {--imagery|--terrain} --config <configfile> -o <output>
   Supported options are:
      --help | -?:     Display this usage message
      --numcpus <num>: Num CPUs to use (default 1)
      --decimation <num>: error threshold for terrain decimation (default 0.009000)